mod-circulation/src/main/resources/domain/mod-circulation/features/loans.feature

```diff
@@ -83,3 +83,158 @@ Feature: Loans tests
     Then status 200
     And match response.itemStatusPriorToCheckIn == 'Checked out'
     And match response.itemId == itemId
+
+  Scenario: Get Loans collection
```

Comment:
```suggestion
  Scenario: When get loans for a patron is called, return a paged collection of loans for that patron with all data as specified in the circulation/loans API
```

rename plz

Reply:
OK, Every time we should name our scenario similar to ticket name, right?

Reply:
At least while we write karate tests based on the documentation